### DISCUSSION TITLE
Installing foundry giving error

### DISCUSSION THREAD

**Discussion by PratikshaGayen** [Cyfrin/foundry-full-course-cu#2836] [General]
PS C:\Blockchain Development\solidity-course> curl -L https://foundry.paradigm.xyz | bash
Invoke-WebRequest : A parameter cannot be found that matches parameter name 'L'.
At line:1 char:6
+ curl -L https://foundry.paradigm.xyz | bash
+      ~~
    + CategoryInfo          : InvalidArgument: (:) [Invoke-WebRequest], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.InvokeWebRequestComm 
   and

**Comment by EngrPips** [comment]
Hello @PratikshaGayen, Please read this post https://github.com/Cyfrin/foundry-full-course-cu/discussions/2518#discussioncomment-10713419 maybe it will help you.

**Reply by PratikshaGayen** [reply]
@EngrPips  I have to install rust first, right?

**Reply by EngrPips** [reply]
If I remember correctly `Foundry` is built with rust, so yeah might need to install rust.
